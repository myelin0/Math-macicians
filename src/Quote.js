const will = 'Mathematics is not about numbers,equations,computations,or algorithms;it is about understanding - William Paul Thurston';
const Quote = () => (
  <p>{will}</p>
);
export default Quote;
