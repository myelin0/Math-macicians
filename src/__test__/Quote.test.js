import React 
import renderer from 'react-test-renderer';
import Quote from "../Quote";
