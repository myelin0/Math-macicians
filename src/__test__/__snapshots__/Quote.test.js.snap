// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<h2
  className="Quoting"
>
  Mathematics is not about numbers,equations,computations,or algorithms;it is about understanding - William Paul Thurston
</h2>
`;
